src/pages/Login/Login.tsx

```diff
@@ -8,19 +8,19 @@ import {
 } from "@ionic/react";
 import React from "react";
 import { useHistory } from "react-router";
-import useAuth from "hooks/useAuth";
+import useAuth from "hooks/useAuth/useAuth";
 
 const Login: React.FC = () => {
-  const { setIsAuthenticated } = useAuth();
+  const { login } = useAuth();
```

Comment:
If you return an array the user will have the choice of naming

```javascript
const [login] = useAuth();
```

Same as `useState`

Just a side note